django/template/defaulttags.py

```diff
@@ -1179,15 +1179,16 @@ def regroup(parser, token):
     As you can see, ``{% regroup %}`` populates a variable with a list of
     objects with ``grouper`` and ``list`` attributes.  ``grouper`` contains the
     item that was grouped by; ``list`` contains the list of objects that share
-    that ``grouper``.  In this case, ``grouper`` would be ``Male``, ``Female``
-    and ``Unknown``, and ``list`` is the list of people with those genders.
+    that ``grouper``.  In this case, ``grouper`` would be ``Guitar``, ``Piano``
+    and ``Trumpet``, and ``list`` is the list of musicians who play this
+    instrument.
 
     Note that ``{% regroup %}`` does not work when the list to be grouped is not
     sorted by the key you are grouping by!  This means that if your list of
-    people was not sorted by gender, you'd need to make sure it is sorted
+    musicians was not sorted by instrument, you'd need to make sure it is sorted
     before using it, i.e.::
 
-        {% regroup people|dictsort:"gender" by gender as grouped %}
+        {% regroup musicians|dictsort:"instrument" by instrument as grouped %}
```

Comment:
Looks good to merge this commit. For the message I'd suggest "Changed the example in the {% regroup %} docstring." to be a bit more descriptive than "Improved".